src/app/index.js

```diff
@@ -16,14 +21,21 @@ const styles = StyleSheet.create({
 // TODO: Pass in the real store to the provider
 const fakeStore = createStore(() => { });
 
-const App = () => {
-  return (
-    <View style={styles.container}>
-      <Provider store={fakeStore}>
-        <DemoButton />
-      </Provider>
-    </View >
-  );
-};
+// const App = () => {
+//   return (
+//     <View style={styles.container}>
+//       <Provider store={fakeStore}>
+//           <Event />
+//       </Provider>
+//     </View >
+//   );
+// };
+
+const App = createBottomTabNavigator({
+  Timeline: { screen: TimelineScreen },
+  Event: { screen: EventScreen },
+  Settings: { screen: SettingsScreen },
+});
 
 export default App;
+console.disableYellowBox = true;
```

Comment:
Should take out this line on master branch, it might swallow some important warning message